### ISSUE TITLE
`<option [value]="...">` results in "WARNING: The bound input value does not exist on any directives or on the element

### ISSUE THREAD

**Issue by zoechi** [dart-lang/angular_analyzer_plugin#199]
```html
<select [(ngModel)]="language"
        (ngModelChange)="update()"
        [disabled]="readonly" class="form-control">
  <option value=""></option>
  <option *ngFor="let lng of languages.keys"
          [value]="lng">foo
 <!--     ^^^^^^^ -->
  </option>
</select>
```
The `<option>` element does have a `value` property https://developer.mozilla.org/en-US/docs/Web/API/HTMLOptionElement

**Comment by MichaelRFairhurst** [comment]
We scan a dart source file for `document.createElement` so that we can see the tagname for that element. Looks like `dialog`, `option`, `media`, `menuitem`, `mod`, `picture` don't use `document.createElement`, and in fact, don't use their respective tag names at any point in the dart source at all.

To make things worse there's `TableSectionElement` which is an interface, not an element at all, yet implements HTMLElement and isn't abstract. I don't know if there's any programmatic way to tell that this isn't a tag.

I will probably hardcode the acceptance of these tags based on their classname and match the tagname by hand...

**Comment by MichaelRFairhurst** [comment]
Merged #214
